### PR TITLE
Feat: Delete charge info workflows

### PR DESCRIPTION
**Pull request by DGajewska**
WATER-3002

* Removes redudant licence methods
* Tweaks charge version workflow methods to work with new model set up
* Adds test data to enable regression tests on charge version workflows

### CODE REVIEW

File: src/modules/charge-versions/controllers/charge-versions.js [hunk 1]
```diff
@@ -62,13 +62,13 @@ const postCreateFromWorkflow = async request => {
   const { chargeVersionWorkflowId } = request.params;
   try {
     const chargeVersionWorkflow = await chargeVersionWorkflowService.getById(chargeVersionWorkflowId);
-
+    console.log({ chargeVersionWorkflow });
```
Comment (Stephan-Hough) [inline src/modules/charge-versions/controllers/charge-versions.js:65]: We don't need this line

File: src/modules/charge-versions/controllers/charge-versions.js [hunk 2]
```diff
@@ -62,13 +62,13 @@ const postCreateFromWorkflow = async request => {
...
     if (!chargeVersionWorkflow) {
       return Boom.notFound(`Charge version workflow ${chargeVersionWorkflowId} not found`);
     }
 
     const approvedBy = userMapper.pojoToModel(request.defra.internalCallingUser);
-
+    console.log({ approvedBy });
```
Comment (Stephan-Hough) [inline src/modules/charge-versions/controllers/charge-versions.js:71]: and this one

**Review by Stephan-Hough — CHANGES_REQUESTED** [review]
There are a couple of console logs that can be removed from src/modules/charge-versions/controllers/charge-versions.js

File: src/modules/charge-versions/controllers/charge-versions.js (hunk 1 above)
Comment (DGajewska) [reply src/modules/charge-versions/controllers/charge-versions.js:65]: Thank you :) removed

File: src/modules/charge-versions/controllers/charge-versions.js (hunk 2 above)
Comment (DGajewska) [reply src/modules/charge-versions/controllers/charge-versions.js:71]: Thank you :) removed

**Review by Stephan-Hough — APPROVED** [review]

**Pull request merged**